src/IterableCodeExtractor.php

```diff
@@ -65,6 +66,30 @@ public static function fromFile( $file_or_files, Translations $translations, arr
 				}
 			}
 
+			if ( ! empty( $options['wpExtractPatterns'] ) ) {
+				$headers = MakePotCommand::get_file_data_from_string(
+					$string,
+					[
+						'Title'       => 'Title',
+						'Description' => 'Description',
+					]
+				);
+
+				if ( ! empty( $headers['Title'] ) ) {
+					$translation = new Translation( 'Title of the pattern', $headers['Title'] );
+					$translation->addReference( $options[ 'file' ] );
+
+					$translations[] = $translation;
+				}
+
+				if ( ! empty( $headers['Description'] ) ) {
+					$translation = new Translation( 'Description of the pattern', $headers['Description'] );
+					$translation->addReference( $options[ 'file' ] );
```

Comment:
Has there been consensus on what to use as the translation context for these new strings?

For `block.json` we use `block title` and `block description` for example, and for `theme.json` `Style variation name`.

So `Pattern title` and `Pattern description` would be more in line with that.

Thoughts?

Reply:
Updated at https://github.com/wp-cli/i18n-command/pull/312/commits/ffa87027ac9ff724d74e454d3d193bfcd7b8903e